docs/index.md

```diff
@@ -1204,6 +1204,13 @@ the statement(s) to move using the left-hand-side check box, then
 move them up and down using the "up" and "down" icons.
 However, there's no need to reorder these statements.
 
+**Important**: Every hypothesis (and goal) id is a database label, so it
+*must* be unique in the database.
```

Comment:
`Every hypothesis (and goal) id is a database label, so it *must* be unique in the database.`

Currently this is not enforced for the goal, but I created an issue for that - https://github.com/expln/metamath-lamp/issues/81